[308] Add onlyIncludeReviewed option to annotations logic

sandrahoang686 opened the pull request:
Ticket: https://github.com/tnc-ca-geo/animl-api/issues/308
Related PRs: https://github.com/tnc-ca-geo/animl-frontend/pull/299

These changes add the `onlyIncludeReviewed` param to the annotations logic.

Self Validation / Demo Locally:
https://www.loom.com/share/242b7d3baecb4b1baa8a855fd286404f?sid=67c61de7-6f70-48a8-af7e-f498796a27d7

sandrahoang686 commented:
Logs / Testing results:

Non-reviewed (109) + Reviewed (2) = 111 total count

**CSV validation output with non-reviewed included**
111 items were outputted
```
_id,originalFileName,path,dateAdded,dateTimeOriginal,cameraId,projectId,make,deploymentId,deploymentName,deploymentTimezone,deploymentLat,deploymentLong,comments,empty,unknown,animal,cougar,coyote,deer,fox,rabbit,rodent,person,bird,other,human,raptor,corvid,skunk,raccoon,dog,bobcat,weasel,Fisher,Black Bear,Red Fox,American Marten,Mouse sp
default_project:bfb17e86f9699acc28b1efb55469f27e,IMG_0150.jpg,100-large-images/IMG_0150.JPG,2025-01-13T17:46:58.961-04:00,2020-02-25T16:06:34.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:39942286f8637c617ebb5317a25e8483,IMG_0356.jpg,100-large-images/IMG_0356.JPG,2025-01-13T17:47:00.005-04:00,2020-02-26T17:04:37.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:e985fc316565e1e542fcdb980397e687,IMG_0160.jpg,100-large-images/IMG_0160.JPG,2025-01-13T17:47:00.302-04:00,2020-02-25T16:24:15.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:051fa9e03f275548db470569e0ff2b20,IMG_0352.jpg,100-large-images/IMG_0352.JPG,2025-01-13T17:47:00.601-04:00,2020-02-26T16:37:01.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:bfdc06a81509552a3e61577273c1f187,IMG_0168.jpg,100-large-images/IMG_0168.JPG,2025-01-13T17:47:00.652-04:00,2020-02-25T16:29:32.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:34f0738ac7ff7be3b46442709c646320,IMG_0361.jpg,100-large-images/IMG_0361.JPG,2025-01-13T17:47:00.766-04:00,2020-02-26T17:15:21.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:28735b0218c46b762f4e291950f33054,IMG_0163.jpg,100-large-images/IMG_0163.JPG,2025-01-13T17:47:00.823-04:00,2020-02-25T16:27:37.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:06e11ef0cea88ab4d98a6de9f351ddb1,IMG_0357.jpg,100-large-images/IMG_0357.JPG,2025-01-13T17:47:00.929-04:00,2020-02-26T17:04:37.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:eac10262393e8423ff5233f0835176c6,IMG_0369.jpg,100-large-images/IMG_0369.JPG,2025-01-13T17:47:00.980-04:00,2020-02-26T17:36:13.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,1,,,,,,,,,,,,,,,,,,,,,,,,
default_project:686ed009e354daedc062de8e4fe29864,IMG_0166.jpg,100-large-images/IMG_0166.JPG,2025-01-13T17:47:01.077-04:00,2020-02-25T16:29:31.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:38af746073bc0bcc0cc5bec46422adc5,IMG_0172.jpg,100-large-images/IMG_0172.JPG,2025-01-13T17:47:01.145-04:00,2020-02-25T16:37:31.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:327530346dbcb8b66291556292aa16a3,IMG_0339.jpg,100-large-images/IMG_0339.JPG,2025-01-13T17:47:01.221-04:00,2020-02-26T15:52:07.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:d3ed0e6c30aba29cc2d3d47b772ebc08,IMG_0153.jpg,100-large-images/IMG_0153.JPG,2025-01-13T17:47:01.270-04:00,2020-02-25T16:09:06.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:0696b2a8991b509a9522ed3cee277132,IMG_0147.jpg,100-large-images/IMG_0147.JPG,2025-01-13T17:47:01.384-04:00,2020-02-25T16:04:45.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:2902b2ee31c4a8cf2f814f261326f677,IMG_0208.jpg,100-large-images/IMG_0208.JPG,2025-01-13T17:47:01.396-04:00,2020-02-25T17:09:35.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:e64f96434ded01ba2d882e8e61fdf561,IMG_0237.jpg,100-large-images/IMG_0237.JPG,2025-01-13T17:47:01.501-04:00,2020-02-25T18:04:39.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:36518d905f2c39a9ef1cfb5f843ae6fc,IMG_0173.jpg,100-large-images/IMG_0173.JPG,2025-01-13T17:47:01.508-04:00,2020-02-25T16:37:31.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:7cbaf52f9a8a7df1aac270c26dc7e205,IMG_0193.jpg,100-large-images/IMG_0193.JPG,2025-01-13T17:47:01.629-04:00,2020-02-25T16:49:16.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:deb0aab9f0aa43faf836149797ed8993,IMG_0335.jpg,100-large-images/IMG_0335.JPG,2025-01-13T17:47:01.651-04:00,2020-02-26T15:49:48.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:4b4354829b9ec070d487d9447449deee,IMG_0338.jpg,100-large-images/IMG_0338.JPG,2025-01-13T17:47:01.755-04:00,2020-02-26T15:52:07.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:15da5f99cd33e580e405b4d4d92e5272,IMG_0149.jpg,100-large-images/IMG_0149.JPG,2025-01-13T17:47:01.776-04:00,2020-02-25T16:06:34.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:a5181d7a4e1d01ad1ed152af4600762b,IMG_0355.jpg,100-large-images/IMG_0355.JPG,2025-01-13T17:47:01.859-04:00,2020-02-26T17:04:37.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:a5fec1cb241f49c6c2f059fa0cdaff70,IMG_0151.jpg,100-large-images/IMG_0151.JPG,2025-01-13T17:47:01.886-04:00,2020-02-25T16:09:05.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:27d712b223022b4aaedd32b30900e1ec,IMG_0337.jpg,100-large-images/IMG_0337.JPG,2025-01-13T17:47:01.971-04:00,2020-02-26T15:52:07.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:6500e3bbaeaf8c378428e49e0aec6364,IMG_0331.jpg,100-large-images/IMG_0331.JPG,2025-01-13T17:47:02.085-04:00,2020-02-26T15:49:03.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:91ab47ded8fb7b6d3f09a3bb31578459,IMG_0162.jpg,100-large-images/IMG_0162.JPG,2025-01-13T17:47:02.205-04:00,2020-02-25T16:24:16.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:9030dd8f367158b49f6589d4b15cc6c0,IMG_0195.jpg,100-large-images/IMG_0195.JPG,2025-01-13T17:47:02.864-04:00,2020-02-25T16:49:17.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:fc296b1b2a2457755905fd5af9f00da8,IMG_0165.jpg,100-large-images/IMG_0165.JPG,2025-01-13T17:47:02.903-04:00,2020-02-25T16:27:37.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:495ec6090151893da7f94cf43f118509,IMG_0232.jpg,100-large-images/IMG_0232.JPG,2025-01-13T17:47:02.976-04:00,2020-02-25T18:03:14.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:4e419fde3d55f7995db2807a02d191ed,IMG_0170.jpg,100-large-images/IMG_0170.JPG,2025-01-13T17:47:03.024-04:00,2020-02-25T16:34:49.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:026a7ba3f2f60ea0d9893426b83fe944,IMG_0202.jpg,100-large-images/IMG_0202.JPG,2025-01-13T17:47:03.078-04:00,2020-02-25T17:02:56.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:471456bd7f9f1b46a2a11f6ded4d0948,IMG_0211.jpg,100-large-images/IMG_0211.JPG,2025-01-13T17:47:03.327-04:00,2020-02-25T17:11:15.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:19bebd836b41dc966acf54ba7a7e5e9e,IMG_0182.jpg,100-large-images/IMG_0182.JPG,2025-01-13T17:47:03.413-04:00,2020-02-25T16:42:08.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:9bd711fe4e8ddca15a308ddee765aa6f,IMG_0340.jpg,100-large-images/IMG_0340.JPG,2025-01-13T17:47:03.444-04:00,2020-02-26T15:53:50.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:fc52888fc5733bf6bde031706eb88ce9,IMG_0234.jpg,100-large-images/IMG_0234.JPG,2025-01-13T17:47:03.553-04:00,2020-02-25T18:03:15.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:f1558c5fb7f513b8d70e6d1dd3964a9a,IMG_0367.jpg,100-large-images/IMG_0367.JPG,2025-01-13T17:47:03.581-04:00,2020-02-26T17:36:13.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:23af8e17ab15d86bd42f0f9e9580f19a,IMG_0159.jpg,100-large-images/IMG_0159.JPG,2025-01-13T17:47:03.597-04:00,2020-02-25T16:13:28.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:9449451616c178db22a41db2a20a7b52,IMG_0229.jpg,100-large-images/IMG_0229.JPG,2025-01-13T17:47:03.657-04:00,2020-02-25T17:44:21.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:bdb96a132942e03e9c57e06a79464798,IMG_0148.jpg,100-large-images/IMG_0148.JPG,2025-01-13T17:47:03.668-04:00,2020-02-25T16:06:33.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:1ac7b1e2603aeb56f0fc3ed3aae9cd00,IMG_0336.jpg,100-large-images/IMG_0336.JPG,2025-01-13T17:47:03.972-04:00,2020-02-26T15:49:49.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:7e96cff2bc7cd7ae3aae5dcff3a2e3ef,IMG_0189.jpg,100-large-images/IMG_0189.JPG,2025-01-13T17:47:04.181-04:00,2020-02-25T16:44:47.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:8190c4f990145b9dbabb98fec05d22c6,IMG_0362.jpg,100-large-images/IMG_0362.JPG,2025-01-13T17:47:04.223-04:00,2020-02-26T17:15:22.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:c43455d60a415d67e1673eb2104b1d5d,IMG_0221.jpg,100-large-images/IMG_0221.JPG,2025-01-13T17:47:04.286-04:00,2020-02-25T17:30:27.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:a7aa1baad51f0e6a62dd2e82012a05ee,IMG_0181.jpg,100-large-images/IMG_0181.JPG,2025-01-13T17:47:04.372-04:00,2020-02-25T16:42:08.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:76d4779ba825fccb620ab4c48e6a3dcc,IMG_0328.jpg,100-large-images/IMG_0328.JPG,2025-01-13T17:47:04.463-04:00,2020-02-26T15:43:21.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:49e4cb5ca14d42c99258517437c25537,IMG_0176.jpg,100-large-images/IMG_0176.JPG,2025-01-13T17:47:04.556-04:00,2020-02-25T16:38:40.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:39e541c09e304041513ff8ca5ee54f85,IMG_0152.jpg,100-large-images/IMG_0152.JPG,2025-01-13T17:47:04.557-04:00,2020-02-25T16:09:05.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:b956492d11f1a6a9f44977624a2986f3,IMG_0368.jpg,100-large-images/IMG_0368.JPG,2025-01-13T17:47:04.631-04:00,2020-02-26T17:36:13.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:978517140767e726c83111696f56b2c6,IMG_0186.jpg,100-large-images/IMG_0186.JPG,2025-01-13T17:47:04.808-04:00,2020-02-25T16:44:15.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:101f4c4ef23afeb6f8e0b6d0542ed708,IMG_0236.jpg,100-large-images/IMG_0236.JPG,2025-01-13T17:47:04.824-04:00,2020-02-25T18:04:39.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:8bce1882071c079ba5d710d0bdff959b,IMG_0158.jpg,100-large-images/IMG_0158.JPG,2025-01-13T17:47:04.852-04:00,2020-02-25T16:13:28.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:e760a578e282881dec75e4a56c2efcc8,IMG_0200.jpg,100-large-images/IMG_0200.JPG,2025-01-13T17:47:04.939-04:00,2020-02-25T17:02:06.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:c5c2a30877ed715166072fb62d474d04,IMG_0175.jpg,100-large-images/IMG_0175.JPG,2025-01-13T17:47:04.937-04:00,2020-02-25T16:38:39.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:364fc3442c8d279b25134ac22b3f7561,IMG_0174.jpg,100-large-images/IMG_0174.JPG,2025-01-13T17:47:05.030-04:00,2020-02-25T16:37:31.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:8b303f72d1ca5a2b54cc814da760d869,IMG_0169.jpg,100-large-images/IMG_0169.JPG,2025-01-13T17:47:05.153-04:00,2020-02-25T16:34:49.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:1508e159b4640bd16a85d4deae01bb99,IMG_0360.jpg,100-large-images/IMG_0360.JPG,2025-01-13T17:47:05.172-04:00,2020-02-26T17:13:11.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:18ea7b20fbb1cca6430eac52c7b3ae73,IMG_0191.jpg,100-large-images/IMG_0191.JPG,2025-01-13T17:47:05.300-04:00,2020-02-25T16:48:42.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:ebaf30bbb56535cf29fbcb6b2bdc1a3b,IMG_0194.jpg,100-large-images/IMG_0194.JPG,2025-01-13T17:47:05.323-04:00,2020-02-25T16:49:17.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:4c06fe9a98e1dcfc2c7a0bedd72bdbbb,IMG_0154.jpg,100-large-images/IMG_0154.JPG,2025-01-13T17:47:05.425-04:00,2020-02-25T16:10:20.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:fd7380885a915fe09f6fe6e4a583c36e,IMG_0230.jpg,100-large-images/IMG_0230.JPG,2025-01-13T17:47:05.426-04:00,2020-02-25T17:44:22.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:8077f85264eaae2e78f2b1cd6e02d0b3,IMG_0187.jpg,100-large-images/IMG_0187.JPG,2025-01-13T17:47:05.540-04:00,2020-02-25T16:44:47.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:db9977b15f6143f7647894e2a3c5b609,IMG_0334.jpg,100-large-images/IMG_0334.JPG,2025-01-13T17:47:05.610-04:00,2020-02-26T15:49:48.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:9c30c4f3b94d9eb11ab7d0540dd18425,IMG_0192.jpg,100-large-images/IMG_0192.JPG,2025-01-13T17:47:05.656-04:00,2020-02-25T16:48:43.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:a89cb6f93cc27761adee5d589d63f00c,IMG_0184.jpg,100-large-images/IMG_0184.JPG,2025-01-13T17:47:05.675-04:00,2020-02-25T16:44:15.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:1460681abeab0ca673dcbb7e690bb0c6,IMG_0341.jpg,100-large-images/IMG_0341.JPG,2025-01-13T17:47:05.675-04:00,2020-02-26T15:53:50.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:dec838644f235d648fedf2e292bfada1,IMG_0342.jpg,100-large-images/IMG_0342.JPG,2025-01-13T17:47:05.738-04:00,2020-02-26T15:53:50.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:79bce440a4cd384cb7d6f6c0c5928c38,IMG_0171.jpg,100-large-images/IMG_0171.JPG,2025-01-13T17:47:05.756-04:00,2020-02-25T16:34:50.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:d301b7bf4503309fc18cae7be2f2f60c,IMG_0161.jpg,100-large-images/IMG_0161.JPG,2025-01-13T17:47:05.785-04:00,2020-02-25T16:24:15.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:7ed5c6c4f8970dbe19f867cf0366fc63,IMG_0199.jpg,100-large-images/IMG_0199.JPG,2025-01-13T17:47:05.815-04:00,2020-02-25T17:02:06.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:8eb776098fe04ab780e961d63706a42a,IMG_0366.jpg,100-large-images/IMG_0366.JPG,2025-01-13T17:47:05.845-04:00,2020-02-26T17:16:28.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:6ce1fca41bbe525fc99c588bc37f7f45,IMG_0210.jpg,100-large-images/IMG_0210.JPG,2025-01-13T17:47:05.849-04:00,2020-02-25T17:09:36.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:f61cf54a925cf092c3b3cebfe46da5a5,IMG_0235.jpg,100-large-images/IMG_0235.JPG,2025-01-13T17:47:05.917-04:00,2020-02-25T18:04:39.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:2cd64d5e13c0b65967a42cd24ecfaaba,IMG_0178.jpg,100-large-images/IMG_0178.JPG,2025-01-13T17:47:05.936-04:00,2020-02-25T16:41:35.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:4f07ab658644e7990c0b78079ef985fc,IMG_0190.jpg,100-large-images/IMG_0190.JPG,2025-01-13T17:47:05.958-04:00,2020-02-25T16:48:42.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:1aa85328a4eace07a4589553b69f06ae,IMG_0156.jpg,100-large-images/IMG_0156.JPG,2025-01-13T17:47:05.958-04:00,2020-02-25T16:10:20.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:b3bc516b271057c76a76b918a4199556,IMG_0164.jpg,100-large-images/IMG_0164.JPG,2025-01-13T17:47:05.985-04:00,2020-02-25T16:27:37.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:e8636ad20b858f17b039c42bbf32d7e6,IMG_0157.jpg,100-large-images/IMG_0157.JPG,2025-01-13T17:47:05.996-04:00,2020-02-25T16:13:28.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:873a66ee8bc31c32638c218ed670c5b4,IMG_0364.jpg,100-large-images/IMG_0364.JPG,2025-01-13T17:47:06.030-04:00,2020-02-26T17:16:27.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:ac91efea2bdb41cd114638b2c54bcc96,IMG_0180.jpg,100-large-images/IMG_0180.JPG,2025-01-13T17:47:06.049-04:00,2020-02-25T16:41:36.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:318b15d6777e1c0f04e8d8fcca97c39b,IMG_0359.jpg,100-large-images/IMG_0359.JPG,2025-01-13T17:47:06.067-04:00,2020-02-26T17:13:11.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:e634199403cdbf3da4d96248efb3a32a,IMG_0204.jpg,100-large-images/IMG_0204.JPG,2025-01-13T17:47:06.085-04:00,2020-02-25T17:02:56.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:d83de872c33ca50254a39bb5941ebd00,IMG_0213.jpg,100-large-images/IMG_0213.JPG,2025-01-13T17:47:06.086-04:00,2020-02-25T17:11:15.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:3677bb4a9d7fa21e0189b144d58e61f0,IMG_0167.jpg,100-large-images/IMG_0167.JPG,2025-01-13T17:47:06.060-04:00,2020-02-25T16:29:31.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:aa799d2667962a87000f7b332a736ff6,IMG_0358.jpg,100-large-images/IMG_0358.JPG,2025-01-13T17:47:06.118-04:00,2020-02-26T17:13:11.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:361ce8ca2804f910a2fefe1eafaaf148,IMG_0233.jpg,100-large-images/IMG_0233.JPG,2025-01-13T17:47:06.112-04:00,2020-02-25T18:03:14.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:cbce30dbfd2630105280b455397b1eae,IMG_0333.jpg,100-large-images/IMG_0333.JPG,2025-01-13T17:47:06.125-04:00,2020-02-26T15:49:04.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:8dd25cb51e5a976a25fb8e900b69b85c,IMG_0220.jpg,100-large-images/IMG_0220.JPG,2025-01-13T17:47:06.152-04:00,2020-02-25T17:30:27.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:fd39d4600c00aa488370477c922d9cce,IMG_0188.jpg,100-large-images/IMG_0188.JPG,2025-01-13T17:47:06.224-04:00,2020-02-25T16:44:47.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:edbbbfa8a4079b8601d1a0b972c0cf69,IMG_0231.jpg,100-large-images/IMG_0231.JPG,2025-01-13T17:47:06.236-04:00,2020-02-25T17:44:22.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:68ecdfd73fed0dedc5587aaa4e800c37,IMG_0146.jpg,100-large-images/IMG_0146.JPG,2025-01-13T17:47:06.305-04:00,2020-02-25T16:04:45.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:10e40e9d27eaa7428fb95d09925d9aac,IMG_0177.jpg,100-large-images/IMG_0177.JPG,2025-01-13T17:47:06.350-04:00,2020-02-25T16:38:40.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:7605b20cbbdce5765e5035ca6a1b912b,IMG_0183.jpg,100-large-images/IMG_0183.JPG,2025-01-13T17:47:06.363-04:00,2020-02-25T16:42:09.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:ca25b6c733ce6dcd54e8f8479baedcc6,IMG_0353.jpg,100-large-images/IMG_0353.JPG,2025-01-13T17:47:06.413-04:00,2020-02-26T16:37:01.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:5a82b84e40e0e92f531f427439889a48,IMG_0155.jpg,100-large-images/IMG_0155.JPG,2025-01-13T17:47:06.421-04:00,2020-02-25T16:10:20.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:5145b68ab0ce3256c646b82734e4c6b6,IMG_0365.jpg,100-large-images/IMG_0365.JPG,2025-01-13T17:47:07.114-04:00,2020-02-26T17:16:27.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:59bc4fcfa7ba9995eb9bcdf4b0724883,IMG_0179.jpg,100-large-images/IMG_0179.JPG,2025-01-13T17:47:07.265-04:00,2020-02-25T16:41:35.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:6ff36ab82f2979893c8c313580caac35,IMG_0222.jpg,100-large-images/IMG_0222.JPG,2025-01-13T17:47:07.293-04:00,2020-02-25T17:30:28.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:dd8834216ae7c6ca098655f945fc1f19,IMG_0363.jpg,100-large-images/IMG_0363.JPG,2025-01-13T17:47:07.320-04:00,2020-02-26T17:15:22.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:6109e2f28bed14efe037837d43dfa87c,IMG_0203.jpg,100-large-images/IMG_0203.JPG,2025-01-13T17:47:07.352-04:00,2020-02-25T17:02:56.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:8ce86a46d7df3955d65b866f9ddca1ea,IMG_0332.jpg,100-large-images/IMG_0332.JPG,2025-01-13T17:47:07.360-04:00,2020-02-26T15:49:03.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:137a831d4c370ba9343fb3194af90106,Dart 13_0254_20220517_154416_976.jpg,DeepFaune-new-england-test-images/Dart 13_0254_20220517_154416_976.JPG,2025-04-17T01:16:43.413-04:00,2022-05-17T18:44:16.000-04:00,dfne,default_project,BUSHNELL,680072885246202c700d2493,dfne-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:5e3fd6071cf120e744267cb68c372b9d,DSCF0017.jpg,DeepFaune-new-england-test-images/DSCF0017.JPG,2025-04-17T01:16:44.200-04:00,2022-05-26T23:56:02.000-04:00,dfne,default_project,unknown,680072885246202c700d2493,dfne-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:03131e938aab27cb0cc8de7094fc7c2f,GMN69_IMG_0058.jpg,DeepFaune-new-england-test-images/GMN69_IMG_0058.JPG,2025-04-17T01:16:44.596-04:00,2024-02-06T13:07:31.000-04:00,dfne,default_project,BROWNING,680072885246202c700d2493,dfne-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:2cc0059a6550715fc83b92dec575dd22,106HSF3MAX_025_20221112_142137_593.jpg,DeepFaune-new-england-test-images/106HSF3MAX_025_20221112_142137_593.JPG,2025-04-17T01:16:44.868-04:00,2022-11-12T18:21:37.000-04:00,dfne,default_project,BROWNING,680072885246202c700d2493,dfne-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:11f7ef52dda335fbaa710b5476668d17,RCNX0048.jpg,seven-rats-nested/rat-folder-1/RCNX0048.JPG,2025-04-17T01:24:33.251-04:00,2021-05-05T03:09:28.000-04:00,dfne,default_project,RECONYX,680072885246202c700d2493,dfne-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:5cdb4dcd23b48a229f01eff14f6049ef,RCNX0052.jpg,seven-rats-nested/rat folder 2/RCNX0052.JPG,2025-04-17T01:24:33.283-04:00,2021-05-05T03:09:31.000-04:00,dfne,default_project,RECONYX,680072885246202c700d2493,dfne-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:9f0ed3c946619386f3d9d7bcb0d02697,RCNX0050.jpg,seven-rats-nested/rat folder 2/RCNX0050.JPG,2025-04-17T01:24:34.415-04:00,2021-05-05T03:09:29.000-04:00,dfne,default_project,RECONYX,680072885246202c700d2493,dfne-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:7bd10ba35a95aa3ef8d1cb698bd12331,RCNX0047.jpg,seven-rats-nested/rat-folder-1/RCNX0047.JPG,2025-04-17T01:24:34.563-04:00,2021-05-05T03:09:27.000-04:00,dfne,default_project,RECONYX,680072885246202c700d2493,dfne-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:74b09ca30a54fbdc43b469a3574de860,RCNX0049.jpg,seven-rats-nested/rat-folder-1/RCNX0049.JPG,2025-04-17T01:24:34.657-04:00,2021-05-05T03:09:29.000-04:00,dfne,default_project,RECONYX,680072885246202c700d2493,dfne-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
default_project:a34925f7ccc9e0fcbbe7ea84ca6524a1,RCNX0046.jpg,seven-rats-nested/rat-folder-1/RCNX0046.JPG,2025-04-17T01:24:35.278-04:00,2021-05-05T03:09:27.000-04:00,dfne,default_project,RECONYX,680072885246202c700d2493,dfne-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,1
default_project:96d74260b319dbf51a5d29316b2aa5d0,RCNX0051.jpg,seven-rats-nested/rat folder 2/RCNX0051.JPG,2025-04-17T01:24:35.382-04:00,2021-05-05T03:09:31.000-04:00,dfne,default_project,RECONYX,680072885246202c700d2493,dfne-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

**CSV validation output with only reviewed**
2 items were outputted
```
_id,originalFileName,path,dateAdded,dateTimeOriginal,cameraId,projectId,make,deploymentId,deploymentName,deploymentTimezone,deploymentLat,deploymentLong,comments,empty,unknown,animal,cougar,coyote,deer,fox,rabbit,rodent,person,bird,other,human,raptor,corvid,skunk,raccoon,dog,bobcat,weasel,Fisher,Black Bear,Red Fox,American Marten,Mouse sp
default_project:eac10262393e8423ff5233f0835176c6,IMG_0369.jpg,100-large-images/IMG_0369.JPG,2025-01-13T17:47:00.980-04:00,2020-02-26T17:36:13.000-04:00,100-large-images,default_project,Prometheus,67854fa17cdda3e7eeb712a2,100-large-images-default,America/Los_Angeles,,,,1,,,,,,,,,,,,,,,,,,,,,,,,
default_project:a34925f7ccc9e0fcbbe7ea84ca6524a1,RCNX0046.jpg,seven-rats-nested/rat-folder-1/RCNX0046.JPG,2025-04-17T01:24:35.278-04:00,2021-05-05T03:09:27.000-04:00,dfne,default_project,RECONYX,680072885246202c700d2493,dfne-default,America/Los_Angeles,,,,,,,,,,,,,,,,,,,,,,,,,,,,1
```



**FirstValidLabel:** 

_Dashboard screenshot_
<img width="1407" alt="Screenshot 2025-04-28 at 4 47 35 PM" src="https://github.com/user-attachments/assets/d8ae19c0-392d-4572-a23a-f000fe215206" />

nathanielrindlaub reviewed (changes requested):
@sandrahoang686, I haven't dug super deep into this but I'm realizing I don't think I properly explained what the goal of this feature is, so let me step back and give you more context. 

The images receive object and label predictions from the ML models, and sometimes users will go through them and validate the predictions or add their own labels that are more accurate. If they do that, the object will get locked, and the "first validated label" in the `Object.labels` array is what we consider the "representative label" for that object. If they do that for all objects in an image, the `Image.isReviewed` flag gets set to `true`.

However, manually reviewing and validating/invalidating/adding new labels is time consuming, and users sometimes want to get a sense of what the ML predictions are saying about their data _without putting in the time to first review the labels manually_. That's really the goal of this feature: we want users to be able to both export and visualize the non-reviewed predictions as well as the reviewed predictions if they so choose. 

So the goal is to find the "representative" label for each object regardless of whether it's locked or whether or not it has validated labels, and then count up all the representative labels for each label class/species. I'm imagining the logic for finding the representative labels will look something like this:

```
const findReprasentativeLabel = (object) => {

// if object is locked and has at least one validated label, return the first validated label in the labels array
// if !onlyIncludeReviewed and the object is unlocked, return the first non-invalidated label in the array
}

Does that make sense? Basically we want to include the representative labels from unlocked objects in the category counts in our output too.

sandrahoang686 commented:
@nathanielrindlaub thanks for the explanation! i believe i've updated the logic to what you were trying to explain. If an image has a locked object, the representativeLabel is the first validated in list (validated is true). If an image has an unlocked object, the representativeLabel is just the first label which is also the most recent label in the list. The image will includes counts of the representedLabel(1) for each object in the image. Does this look/sound correct?

I also updated the COCOAnnotation logic because I didn't do that before since the [ticket](https://github.com/tnc-ca-geo/animl-api/issues/308#issue-2920990786) only mentioned updating `flattenImgTransform` which doesnt happen when creating the COCOAnnotation. Lmk if I shouldn't have updated this though 👍🏼

lessej commented on src/task/annotations.ts:
```diff
@@ -70,7 +72,7 @@ export class AnnotationsExport {
     this.filename = `${documentId}_${format}${this.ext}`;
     this.bucket = config['/EXPORTS/EXPORTED_DATA_BUCKET'];
-    this.onlyIncludeReviewed = true; // TODO: move into config or expose as option?
+    this.onlyIncludeReviewed = onlyIncludeReviewed; // TODO: move into config or expose as option?
```
Probably can remove this TODO because it's completed with this PR

lessej commented on src/@types/graphql.ts:
```diff
@@ -373,6 +373,7 @@ export type ExportErrorsInput = {
 export type ExportInput = {
   filters: FiltersInput;
   format: Format;
+  onlyIncludeReviewed?: InputMaybe<Scalars['Boolean']['input']>;
```
On the frontend side, I don't seen anywhere where this is not passed into the request.  Is there a reason this should be optional?

sandrahoang686 replied:
no this could be marked as required, i think i did this before making the FE changes and just wanted insurance in case if it wasn't passed in. removing 👍🏼

nathanielrindlaub commented on src/task/annotations.ts:
```diff
@@ -86,15 +88,20 @@ export class AnnotationsExport {
       }
 
       this.pipeline = buildPipeline(sanitizedFilters, this.projectId);
-      this.imageCount = await this.getCount(this.pipeline);
```
I think this might have been an accident. I put it back in.

sandrahoang686 replied:
i removed that by accident, thanks for adding back in

nathanielrindlaub reviewed (approved):
@sandrahoang686 I made a few small changes but this is looking good. If you don't mind reviewing the commits I made and letting me know if any were unclear just so we're on the same page, that would be great! Merging!
